studio/schemas/toy.ts

```diff
@@ -0,0 +1,147 @@
+const baseObject = { name: "", type: "boolean", title: "" };
```

Comment:
@LPRegen tambien le puedes agregar estos valores, es para que se muestren todos los campos al hacer los fetchs

```suggestion
const baseObject = {
  title: "",
  name: "",
  type: "boolean",
  initialValue: false,
  options: { layout: "checkbox" },
};
```